p/polandscript.pls

```diff
@@ -0,0 +1,4 @@
+//hello world program in polandscrpt
```

Comment:
Can you send a link to this programming language?

Reply:
no